onnxruntime/test/optimizer/transpose_optimizer_test.cc

```diff
@@ -291,210 +291,6 @@ TEST(TransposeOptimizerTests, TestPadNonconst) {
                     /*opset_version*/ 11);
 }
 
-TEST(TransposeOptimizerTests, TestResize) {
```

Comment:
Why are other unit tests being deleted?

Reply:
They are cases testing the optimizer to be removed here.